das_api/src/builder.rs

```diff
@@ -94,6 +94,17 @@ impl RpcApiBuilder {
         )?;
         module.register_alias("getAssetsByGroup", "get_assets_by_group")?;
 
+        module.register_async_method(
```

Comment:
I believe the other maintainers wanted an alias to be added? @danenbm do you still need this?

Reply:
I think other maintainers did strongly advocate for `getAssetProofs`, so it does make sense to add `getAssetSignatures` to match that API format that we probably want to standardize.

Even though I understand that `getSignaturesForAsset` follows a Solana RPC ("getSignaturesForAddress") format.

Reply:
sure, adding an alias doesn't matter much to me